tests/tests/eth_provider.rs

```diff
@@ -1355,21 +1333,22 @@ async fn test_transaction_by_hash(#[future] katana: Katana, _setup: ()) {
         .await
         .expect("failed to send transaction");
 
-    // Retrieve the pending transaction from the database
-    let mut stored_transaction: StoredPendingTransaction =
-        eth_provider.database().get_first().await.expect("Failed to get transaction").unwrap();
+    // TODO: need to write this with the mempool
```

Comment:
I think you can just uncomment and ignore the test no?

Reply:
Mhh don't think this is possible because here we previously created a `stored_transaction` at the beginning which has `StoredPendingTransaction` type and the rest of the test depends on it. But now we don't have this type anymore so this won't compile. I think this is the same situation for the other places.